bin/css-validator

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+`echo eval exec node $(printf %q\  "$0" "$@")`
+const fs = require('fs')
+const validateCss = require('css-validator')
+
+validateCss({text: fs.readFileSync(process.argv[2])}, (err, data) => {
+  if (err) console.error(err)
+  if (data.errors.length) console.error(data.errors)
+  if (data.warnings.length) console.error(data.warnings)
```

Comment:
This is definitely a can of worms for formatting. Let's revisit this after we get the CLI parsing sorted out. For reference, here's what I do in my `restructuredtext-lint` repo:

https://github.com/twolfson/restructuredtext-lint/blob/1.0.1/restructuredtext_lint/cli.py#L45-L67